src/components/Inputs.tsx

```diff
@@ -193,13 +194,30 @@ const Inputs: React.FC = () => {
             onClick={toggleSecretVisibility}
           />
         </div>
+        {addNewClaim ? (
```

Comment:
This logic should be placed in a separate branch/pull request. "feature/add-dynamic-claims" or something. From the name of this PR, it should only contain refactorings ( code improvements ). 

It should also be possible to add multiple claims. Or, if only one claim is supported the "Add claim" button should be disabled when a claim is added.

Reply:
Okay, I will try to fix that, thanks!